tests/unit/src/test/scala/tests/NewScalaWorksheetLspSuite.scala

```diff
@@ -0,0 +1,25 @@
+package tests
+
+import scala.meta.internal.metals.ServerCommands
+import scala.meta.internal.metals.MetalsEnrichments._
+
+object NewScalaWorksheetLspSuite extends BaseLspSuite("new-scala-worksheet") {
```

Comment:
Heads up, when rebasing on top of master you will need to change this `object` into a `class`. See #1277